plugin/src/resolve-gatsby-image-data.ts

```diff
@@ -29,11 +51,14 @@ export async function resolveGatsbyImageData(
     filename,
     width,
     height,
-    toFormat
-  ): IImage => {
+    toFormat,
+    _fit,
+    _options
```

Comment:
Do we need to add these parameters if we're just going to ignore them?

Reply:
Nah, I just forgot to remove these - I was looking into them when I was testing and wasn't sure how to get the options stuff figured out (was mostly just exploring and forgot to cleanup). Will remove!